mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -935,8 +910,8 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
         propsUpdateHandle = null
     }
 
-    private fun manageHabpanelShortcut(visible: Boolean) {
-        manageShortcut(visible, "habpanel", ACTION_HABPANEL_SELECTED,
+    private fun manageHabPanelShortcut(visible: Boolean) {
+        manageShortcut(visible, "habPanel", ACTION_HABPANEL_SELECTED,
```

Comment:
If this is changed, the HABPanel shortcut will be duplicated.

Reply:
Oops, this change wasn't intentional.